readme.md

```diff
@@ -45,31 +45,70 @@ npm test
 
 Need help? Check out our docs [here](https://stenciljs.com/docs/my-first-component).
 
-
 ## Naming Components
 
 When creating new component tags, we recommend _not_ using `stencil` in the component name (ex: `<stencil-datepicker>`). This is because the generated component has little to nothing to do with Stencil; it's just a web component!
 
-Instead, use a prefix that fits your company or any name for a group of related components. For example, all of the Ionic generated web components use the prefix `ion`.
-
+Instead, use a prefix that fits your company or any name for a group of related components. For example, all of the Ionic-generated web components use the prefix `ion`.
 
 ## Using this component
 
-There are three strategies we recommend for using web components built with Stencil.
+There are two strategies we recommend for using web components built with Stencil.
 
 The first step for all three of these strategies is to [publish to NPM](https://docs.npmjs.com/getting-started/publishing-npm-packages).
```

Comment:
Given the changes made in this PR, I'm not sure this sentence makes total/complete sense. I think that it's valuable to clarify for something like unpkg you'd wanna publish your stuff, so maybe we just change 'three' to 'two' here?
```suggestion
The first step for all two of these strategies is to [publish to NPM](https://docs.npmjs.com/getting-started/publishing-npm-packages).
```